app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,39 @@
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def remember(user)
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
```

Comment:
__[reek]__
* [FeatureEnvy] refers to user more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)